detailed_docs/TRS/RC/GetInteriorVehicleDataConsent.md

```diff
@@ -2,23 +2,61 @@
 
 1.
 In case
-- SDL received OnRemoteControlSettings notification from HMI with "ASK_DRIVER" access mode
-- and RC application (in HMILevel FULL) requested access to remote control module that is already allocated to another RC application
-- and SDL requested user consent from HMI via GetInteriorVehicleDataConsent
-- and user allowed access to RC module for requested application
+- SDL received OnRemoteControlSettings notification from HMI with `accessMode=ASK_DRIVER`
```

Comment:
@KhrystynaDubovyk Please add next conditions into "In case" section according last changes in GetInteriorVehicleDataConsent implementation:
```
 - SDL received SetGlobalProperties request from mobile with userLocation which is within serviceArea of each requested in GetInteriorVehicleDataConsent modules or it is driver location
 - Requested in GetInteriorVehicleDataConsent modules are already allocated to another RC application
 - Requested in GetInteriorVehicleDataConsent modules have allowMultipleAccess = true
```